tests/integration/mounts/test_mounts.py

```diff
@@ -0,0 +1,29 @@
+from databricks.labs.ucx.mixins.compute import CommandExecutor
+from databricks.labs.ucx.mounts.list_mounts import Mount
+
+
+def test_mount_listing(ws, wsfs_wheel, make_schema, sql_fetch_all):
+    # TODO: create a mount point in a freshly created bucket instead of leveraging existing mount point
```

Comment:
add `pytest.skip("needs mountpoints")` + create an issue in the repo to require testing mountpoints